analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -128,9 +138,13 @@ class Maven : PackageManager() {
 
         val vcsFromPackage = maven.parseVcsInfo(mavenProject)
 
+        // If running in SBT mode expect that POM files were generated in a "target" subdirectory and that the correct
+        // project directory is the parent of this.
+        val projectDir = if (sbtMode) workingDir.parentFile else workingDir
+
         val project = Project(
                 id = Identifier(
-                        packageManager = javaClass.simpleName,
+                        packageManager = if (sbtMode) SBT::class.java.simpleName else javaClass.simpleName,
```

Comment:
Hmm, I have another proposal how to fix this in a branch of mine (currently called "maven-all", but I'll rename it). Mine requires more code, but in exchange does not add a dependency on SBT from Maven. My plan was to change `resolveDependencies()` in `SBT.kt` like so:

    override fun resolveDependencies(projectDir: File, definitionFiles: List<File>): ResolutionResult {
        val result = mutableMapOf<File, AnalyzerResult>()

        // Simply pass on the list of POM files to Maven, ignoring the SBT build files here.
        Maven.create().resolveDependencies(projectDir, prepareResolution(definitionFiles)).mapValuesTo(result) {
            val sbtProjectId = it.value.project.id.copy(packageManager = javaClass.simpleName)
            val sbtProject = it.value.project.copy(id = sbtProjectId)
            it.value.copy(project = sbtProject)
        }

        return result
    }

Reply:
I needed the SBT dependency anyway in another place, so I also used it here. I would prefer to keep it here as is to be able to merge this, but feel free to change it again in your branch.